Makefile

```
@@ -0,0 +1,52 @@
+APP = triggers
+
+IMAGE_BASE ?= ghcr.io/shipwright-io
+IMAGE_TAG ?= latest
+
+GOFLAGS ?= -v -mod=vendor
+GOFLAGS_TEST ?= -v -cover -race
+
+NAMESPACE ?= shipwright-build
+
+KO_DOCKER_REPO ?= $(IMAGE_BASE)
+KO_OPTS ?= --base-import-paths --tags=${IMAGE_TAG}
+
+.EXPORT_ALL_VARIABLES:
+
+.PHONY: $(APP)
+$(APP):
+	go build .
```

```suggestion
	go build . -race
```
Might be worth adding the `-race` flag